packages/store/addon/-private/system/core-store.ts

```diff
@@ -270,12 +269,12 @@ abstract class CoreStore extends Service {
     ```js
     import Store from '@ember-data/store';
 
-    export default Store.extend({
+    export default class MyStore extends Store {
       init() {
         this._super(...arguments);
```

Comment:
`super.init(...arguments);`

Reply:
I also missed converting the init() to constructor(). Fixed